src/freedv_api.h

```diff
@@ -75,6 +75,8 @@
 #define FREEDV_SYNC_AUTO   1                 // falls out of sync automatically
 #define FREEDV_SYNC_MANUAL 2                 // fall out of sync only under operator control
 
+#define FREEDV_VARICODE_MAX_BITS  12         // max bits for each ASCII character
```

Comment:
Would this be better as a function? That said, I don't foresee this changing any time soon, either.

Reply:
Yes I did consider that, but arrived at the same conclusion.